docs/rule/no-bare-strings.md

```diff
@@ -22,32 +22,20 @@ This rule **allows** the following:
  The following values are valid configuration:
 
 * boolean -- `true` for enabled / `false` for disabled
-* array -- an array of allowlisted strings
+* array -- An array of allowlisted strings
 * object -- An object with the following keys:
   * `allowlist` -- An array of allowlisted strings
   * `globalAttributes` -- An array of attributes to check on every element.
   * `elementAttributes` -- An object whose keys are tag names and value is an array of attributes to check for that tag name.
+  * `ignoreDefaultAllowList` -- A boolean to optionaly ignore default allowlist
```

Comment:
How about we actually call this `includeDefaultAllowList` (same change for other new options too)? That removes the negative--I usually try to avoid negatives in option names. Also we will want to make sure we mention the default value here.